### ISSUE TITLE
keyboard not displayed after 2.6 update

### ISSUE THREAD

**Issue by ayalalazaro** [dixonary/fingerterm-reMarkable#8]
After the new 2.6 remarkable update the keyboard is no longer displayed
It throws the following error:
"Creating window
virtual void EPRenderContext::renderNextFrame(QSGRenderer*, uint) can't render without a renderer
Segmentation fault"

Probably due to a change in toolchain as described here [https://github.com/toltec-dev/toltec/issues/322](https://github.com/toltec-dev/toltec/issues/322)

**Comment by sorenisanerd** [comment]
I tried rebuilding fingerterm with the new toolchain. No dice. @matteodelabre, did you actually get it working with the new toolchain? It's a slightly different error, but it's still not functional.

**Comment by matteodelabre** [comment]
You’re right @sorenisanerd, it’s not working. Up to this point, I had only tested the new toolchain with the simpler [calculator app](https://github.com/reHackable/Calculator) (which works fine with the new toolchain). fingerterm must be doing something different that is causing the crash. I’ll investigate this.

**Comment by matteodelabre** [comment]
@sorenisanerd Should be fixed with toolchain v2.0.1

**Comment by ard0gg** [comment]
Can confirm fingerterm is fixed in toolchain v2.0.1

**Comment by darkliquid** [comment]
are new builds available in toltec testing yet or is a new build of the binary available to download?

**Comment by matteodelabre** [comment]
FYI: Toltec testing now provides a version of fingerterm that is compatible with releases ⩾2.6.
